packages/tools/kadena-cli/src/tx/utils/txHelpers.ts

```diff
@@ -81,18 +72,18 @@ export async function getTransactions(
           const filePath = join(directory, fileName);
           const content = await services.filesystem.readFile(filePath);
           if (content === null) return null;
-          // signed=false can still return already signed transactions
-          const schema = signed ? ICommandSchema : IUnsignedCommandSchema;
-          const JSONParsedContent = JSON.parse(content) as
-            | ICommand
-            | IUnsignedCommand;
-          const parsed = schema.safeParse(JSONParsedContent);
-          if (parsed.success && !signed && !all) {
-            const isSigned = JSONParsedContent.sigs.every((sig) => !!sig);
-            if (isSigned) return null;
+          const JSONParsedContent = JSON.parse(content);
+          const isSignedTx =
+            'sigs' in JSONParsedContent &&
```

Comment:
this is incorrect, beause a transaction can also be partially signed, where it has a sigs array with a sig, and null for example if there was one signed and one unsigned and that would still make it unsigned.

Reply:
yes, you're right! partially signed transaction needs to be considered as unsignedso that when user wants to sign with another set of key will still list them as part of sign command. 

Even when we're running `tx list` that is the current behaviour partial signed transaction considered as unsigned as well. 

in case if i'm missing we can have a quick call and discuss it

Reply:
Hmm.. I'm wondering if using the schemas like this still makes sense if the method needs to return all transactions anyway.

I had set up the two schemas as a filter between signed and unsigned, but since you are now accepting all transactions I don't think that's needed anymore.

If you already used `isSignedTransaction` there is no extra value to using the `ICommandSchema` over the `IUnsignedCommandSchema`, this is just double-checking the same thing right?